website/blog/2020-07-01-how-to-create-near-real-time-models-with-just-dbt-sql.md

```diff
@@ -13,6 +13,13 @@ date: 2020-07-01
 is_featured: false
 ---
 
+:::caution More up-to-date information available
+
+The information contained in this blog post is outdated! You should now refer to the most recent blog post [about materialized views](/blog/announcing-materialized-views) or our official documentation about [materializations](/docs/build/materializations).
```

Comment:
```suggestion
Since this blog post was first published, many data platforms have added support for [materialized views](/blog/announcing-materialized-views), which are a superior way to achieve the goals outlined here. We recommend them over the below approach. 
```